function-composition/README.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Function Composer
+category: Behavioral
+language: en
+tag:
+  - Functional Programming
+  - Composition
+  - Java
```

Comment:
Prefer tags from the list https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags